src/eva/tests/config/testMonSummary.yaml

```diff
@@ -12,12 +12,11 @@ datasets:
     sensor: hirs4
     control_file:
       - ${data_input_path}/time.hirs4_metop-a.ctl
-    filenames:
-      - ${data_input_path}/time.hirs4_metop-a.2015051418.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051500.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051506.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051512.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051518.ieee_d
+    filenames_template:
+        template: ${data_input_path}/time.hirs4_metop-a.%Y%m%d%H.ieee_d
+        start: "2015051418"
```

Comment:
What are people's thoughts on using the ISO time stamp format instead for these?
```
start: 2015-05-14T18:00:00Z
```